src/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ add_library(gse
 
 set_target_properties(gse
     PROPERTIES
-        CXX_STANDARD 17
+        CXX_STANDARD 20
```

Comment:
Is everyone dependent on this code OK with the move to C++20?

Reply:
No